raiden/network/transport/matrix/transport.py

```diff
@@ -579,8 +583,7 @@ def stop(self) -> None:
         self._stop_event.set()
         self._broadcast_event.set()
 
-        if self._raiden_service:
-            self._web_rtc_manager.stop()
+        self._web_rtc_manager.stop()
```

Comment:
I think the condition was there for a reason (although it was a false condition). Now, with the change that the web_rtc_manager is only initialized in `start()`, the argument for the if condition is even stronger.

If `stop()` is called before `start()` then `self.web_rtc_manager` will be none. I wonder why the linter does not prevent that.

Reply:
If `stop` is called before `start`, it will exit immediately because of the `_stop_event` check. So it will all be good. In fact, I'm going to explicitly reset `_web_rtc_manager` to `None` to make it even more clear that it's valid only in the time period `<start, stop>`.